gb-cpu/src/soc.rs

```diff
@@ -0,0 +1 @@
+pub mod cpu;
```

Comment:
I'm thinking if shouldn't be outside of the `cpu` crate since it will contain the `cpu`, `ppu` and maybe the `bus` ?

Reply:
Yes, your're absolutly right, I'm thinking this should be in the main of the project